src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/FilesTabViewImpl.java

```diff
@@ -39,13 +40,13 @@ public class FilesTabViewImpl implements FilesTabView {
   SimplePanel addToDownloadListWidgetContainer;
 
   @UiField
-  Column filePreviewContainer;
+  Grid filePreviewContainer;
 
   @UiField
   Div filePreviewWidgetContainer;
 
   @UiField
-  Column fileProvenanceContainer;
+  FlowPanel fileProvenanceContainer;
```

Comment:
I replaced a lot of grids with one item & no responsive traits (i.e. xs = 12) with a `FlowPanel`.